plugins/plugin-docker/che-plugin-openshift-client/src/main/java/org/eclipse/che/plugin/openshift/client/OpenShiftConnector.java

```diff
@@ -315,6 +316,7 @@ public ContainerCreated createContainer(CreateContainerParams createContainerPar
                                                  .withName(imageStreamName)
                                                  .get();
         if (imageStream == null) {
+            openShiftClient.close();
```

Comment:
@amisevsk oh, sorry overlook try block. Still I think that client should be opened only once and closed only once (in finally) in each method. So, maybe it makes sense to wrap the whole method body in try ?